process/reference-architectures.md

```diff
@@ -0,0 +1,56 @@
+# Reference Architecture Submission Process
+
+This document summarized the process for the submission, review and publication of new reference architectures in the CNCF. It describes the expected workflow and the different stakeholders involved in each step.
+
+![Reference Architecture Workflow Diagram](reference-architectures-workflow.png)
+
+## Workflow Steps
+
+### Proposed
+
+*Who: Individual End Users, Groups of End Users, Community Groups*
+
+New reference architectures should be submitted via [this github template]() (TODO template!) by individual end users, groups of end users from a specific domain or end user group, or community groups.
```

Comment:
@mrbobbytables @taylorwaggoner you work on the template magic or should i do it?

Reply:
Being worked here: https://github.com/cncf/tab/pull/73